lib/config/rc2014.cfg

```diff
@@ -11,17 +11,23 @@ CRT0     DESTDIR/lib/rc2014_crt0
 OPTIONS  -O2 -SO2 -iquote. -D__Z88DK -D__RC2014 -D__Z80 -clib=sdcc_iy -subtype=basic
 
 CLIB     default -lrc2014_clib -lndos -Cc-standard-escape-chars
+
+CLIB     rc2014-8085 -Cc-standard-escape-chars -D__SCCZ80 -D__8085 -D__8085__ -l8085_opt -lndos -lrc2014-8085_clib
```

Comment:
I think this needs to have -startuplib=8080_crt0 ?

Reply:
I'll add it. I nearly threw all my toys out of the cot this afternoon, but perhaps some progress now.